### DISCUSSION TITLE
Welcome to the OpenMined Courses discussion board!

### DISCUSSION THREAD

**Discussion by yemikifouly** [OpenMined/courses#2] [Announcements]
Hi there! 👋

Welcome to the **[OpenMined Courses](https://courses.openmined.org/)** discussion board!

You will use this board to post questions you have about the courses content, receive technical support from our mentors, share your thoughts about the courses and mingle with your fellow students.

Welcome once again. Please, follow these steps to get started! 🙂

## Step 1. Log in

If you haven't yet, log in to your GitHub account. You must be logged in to post.

## Step 2. Read the Code of Conduct

If you haven't yet, please go through the **[Code of Conduct](https://github.com/OpenMined/courses/blob/master/CODE_OF_CONDUCT.md)**. Follow the code to help us build a strong community.

## Step 3. Discover the board

The discussion board has 7 categories.

**One** category is for us alone to post under:

 * **[📢 Announcements](https://github.com/OpenMined/courses/discussions/categories/announcements)**: Announcements about the courses and courses-related events. 

> ⚠️ **Even though you technically can, please DO NOT post under "Announcements".**

**Two** categories can be used to post about the course:

* **[🗣️ Course: Our Privacy Opportunity](https://github.com/OpenMined/courses/discussions/categories/course-our-privacy-opportunity)**: Questions related to the content of the "Our Privacy Opportunity" course.
* **[📋 Project: Privacy Product Specification](https://github.com/OpenMined/courses/discussions/categories/project-privacy-product-specification)**: Questions related to the "Privacy Project Specification" project.

**Four** categories for open-ended discussions:

* **[🔄 Feedback](https://github.com/OpenMined/courses/discussions/categories/feedback)**: Feedback about the courses, the projects, the support services...
* **[💬 General](https://github.com/OpenMined/courses/discussions/categories/general)**: Open-ended discussions about anything and everything.
* **[🤝 Introduce Yourself](https://github.com/OpenMined/courses/discussions/categories/introduce-yourself)**: Introductions...  
* **[📚 Resources](https://github.com/OpenMined/courses/discussions/categories/resources)**: Share resources, lessons summaries, tips and advice... 

> ⚠️ **To report a bug on the website, you should NOT post under "Feedback". Instead, raise an issue [here](https://github.com/OpenMined/openmined/issues).**

## Step 4. Introduce yourself

Get acquainted with the courses staff and your peers by scrolling through the posts under **[Introduce Yourself](https://github.com/OpenMined/courses/discussions/categories/introduce-yourself)**. Then, post an introduction yourself to tell us more about you.

## Step 5: Check out our FAQs

Check out **[our FAQs](https://github.com/OpenMined/courses/discussions/76)** to get answers to your general questions. 

From everyone at OpenMined, welcome to the OpenMined Courses discussion board! 🙂

**Comment by mcleonard** [comment]
Hi Yemi! Happy to be here.

**Comment by em-blue** [comment]
Super cool! Hi Yemi and Mat!

**Comment by ghost** [comment]
Hello!

**Reply by iamtrask** [reply]
Hello!
